storage/migrations/08_agg_stats_refactor.up.sql

```diff
@@ -0,0 +1,49 @@
+-- Drop stats materialized views and replace with tables.
+-- Also migrate the data from the old materialized views to the new tables.
```

Comment:
I think it's nice to support already computed values and since it's pretty straightforward i'd be keen in keeping it in.

> re about greppability:

Agree, it reduces it, but I think we should aim for a general solution to improve the greppability of the DB schema and migrations. One possible approach would be that every so many migrations (e.g. maybe at 10), we would flatten and collapse all migrations into the initial (3? consensus/runtime/stats?) migrations and remove the other migration files.

Reply:
> Whoever wanted to deploy the "flattened" migrations would first have to make sure they've deployed the release just before that so that their DB schema is the newest version

I actually imagined this differently. The "flattened" migrations would work with empty database. And existing deployments would just skip the "flattened" migrations.

Reply:
Regardless, I decided to drop the migration because I figured that the migration should also handle the case where the last daily window can actually be incomplete. Also didn't test the time it would take to do the migration on staging/prod.

Instead, I updated the main aggregate stats worker code to detect the case where it is catching up on the stats (if the per-iteration batch limit is reached) and in that case use a very short timeout before recomputing the stats again (instead of waiting 1 minute). See last commit.